bender_msgs/CylindricalObject target
---
trajectory_msgs/JointTrajectoryPoint[] points
float64[] costs
string[] joint_names